tests/ga/test_update.py

```diff
@@ -1086,11 +1086,11 @@ def test_run_latest(self):
         args = args[0]
         cmds = textutil.safe_shlex_split(agent.get_agent_cmd())
         if cmds[0].lower() == "python":
-            cmds[0] = get_python_cmd()
+            cmds[0] = get_complete_python_cmd()
 
         self.assertEqual(args, cmds)
         self.assertTrue(len(args) > 1)
-        self.assertTrue(args[0].startswith("python"))
+        self.assertRegex(args[0], r"(/*/python\d*)$", "The command doesn't contain full python path")
```

Comment:
This regex doesn't seem right; `/*/` is equivalent to `/+` and matches one or more `/` in a row. Seems like we're missing a `.`? Aka `/.*/python\d*`.

Additionally, if we're only wanting to match one or zero digits at the end, `\d?` might be a better option.

Reply:
That is a very good point, I wonder how this is working lol (I am clearly not a regex expert lol). Let me play around with it to see what's happening there.

>Additionally, if we're only wanting to match one or zero digits at the end, \d? might be a better option.
Sometimes the interpreter can also be something like `/usr/bin/python3.6` or something, essentially not necessary that it'll always be just a digit.

Reply:
I just realized the regex doesn't work for `python3.6` either lol. Let me fix that too.

Reply:
Fixed

Reply:
Lol sure, didn't realize that was a possible name for the python exec. I think the original was working because the regex is just doing a search instead of a match all, i.e. `.../python3` is matched, but not the whole path.

Reply:
Yes that is absolutely correct, good catch! I know who to come to now for regex related issues :)

Reply:
Always happy to help!